staging/src/k8s.io/component-base/logs/logs.go

```diff
@@ -76,3 +82,82 @@ func GlogSetter(val string) (string, error) {
 	}
 	return fmt.Sprintf("successfully set klog.logging.verbosity to %s", val), nil
 }
+
+const logLoggingFormat = "logging-format"
+
+var logRegistry *LogFormatRegistry = NewLogFormatRegistry()
```

Comment:
Could you move registry code to separate file `registry.go`, and options struct to `options.go`?